sdk/repo.go

```diff
@@ -153,5 +156,9 @@ func RepoSync(chroot string, force bool) error {
 	if force {
 		args = append(args, "--force-sync")
 	}
-	return enterChroot(chroot, chrootRepoRoot, args...)
+	return enterChroot(enter{
+		Chroot: filepath.Join(RepoRoot(), chroot),
```

Comment:
This still has the `Join` in all of the callers.  My point was that `enterChroot` could do it before marshalling.